ansible/playbooks/AdoptOpenJDK_Unix_Playbook/trss.yml

```diff
@@ -23,18 +23,12 @@
           apt: upgrade=safe update_cache=yes
           tags: patch_update
 
-        - name: Download Nodev12.16.1 tarball
-          get_url:
-            url: https://nodejs.org/dist/v12.16.1/node-v12.16.1-linux-x64.tar.xz
-            dest: /tmp/node-v12.16.1-linux-x64.tar.xz
-            mode: 0440
-          tags: node_install
-
-        - name: Extract Node tarball
+        # Prepare for node.js
+        - name: Download Node v12.16.1 tarball and extract Node tarball
```

Comment:
Do we need a newer node version?

Reply:
we can, but i would prefer having another PR to upgrade node version.
at least need to test with trss first

Reply:
sure, will create a new issue for upgrade

Reply:
FYI: https://github.com/adoptium/infrastructure/issues/2591

Reply:
@karianna Are you ok for this to go in now and the node update evaluated in a separate PR. We'll be doing some changes relating to how TRSS is deployed in the next few weeks so I suspect we'll look at the Node upgrade alongside that work.

Reply:
@karianna can you have another review on this PR, thanks!